jaeger-core/src/test/java/io/jaegertracing/internal/propagation/TextMapCodecTest.java

```diff
@@ -158,4 +158,45 @@ public void testAdhocBaggageWithoutTraceId() {
     assertEquals("v2", context.getBaggageItem("k2"));
     assertEquals(null, context.getBaggageItem("k3"));
   }
+
+  @Test
+  public void testInjectDoNotEncodeSpanContext() {
+    TextMapCodec codec = new TextMapCodec(true);
+    Map<String, String> headers = new HashMap<String, String>();
+    long traceIdLow = 42;
+    long spanId = 1;
+    long parentId = 0;
+    codec.inject(new JaegerSpanContext(0L, traceIdLow, spanId, parentId, (byte)1), new TextMapAdapter(headers));
+
+    String traceId = headers.get("uber-trace-id");
+    assertEquals("2a:1:0:1", traceId);
+  }
+
+  @Test
+  public void testExtractSupportEncodedSpanContext() {
```

Comment:
I think this and next method names should be swapped

Reply:
Oops, fixed.